internal/envconfig/envconfig.go

```diff
@@ -59,6 +59,7 @@ var (
 func init() {
 	switch strings.ToLower(os.Getenv(requireHandshakeStr)) {
 	case "on":
+	default:
```

Comment:
This switch statement doesn't fall through. `case "on"` has no actions associated with it.